lektor/admin/static/js/widgets.jsx

```diff
@@ -75,6 +77,7 @@ class FieldBox extends Component {
             type={field.type}
             placeholder={placeholder}
             disabled={disabled}
+            {...this.getRoutingProps()}
```

Comment:
I assume this is needed because of the Image/Link widgets?

It might be a lot of work, but maybe it's possible to pass down routing information using the context API instead?

https://react-legacy.netlify.com/docs/context.html (should be applicable to React version Lektor uses)

Reply:
While it would be possible to do this, it would make the way a FieldBox passes the routing props around be different to how they're passed around in the reset of the admin UI. Rather than changing it here, it may be worth opening a new issue and changing it throughout the entire UI instead? I'm open to doing it here if that's what is preferred though.

Reply:
I'm fine with that, keep the way you have.